content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1141,6 +1472,17 @@ Defines a [default value for a field](../../../concepts/components/prisma-schema
 - Default values that cannot yet be represented in the Prisma schema are represented by the `dbgenerated()` function when you use [introspection](../../concepts/components/introspection).
 - Default values are currently not allowed on relation fields in the Prisma schema. Note however that you can still define default values _manually_ in the underlying database using plain SQL. These will not be represented in the Prisma schema though, this is a temporary limitation which you can [track on GitHub](https://github.com/prisma/prisma/issues/1111).
 
+##### MongoDB
+
+- Default values can be a static value (`4`, `"hello"`) or one of the following [functions](#attribute-functions):
+
+  - [`dbgenerated()`](#dbgenerated) (can only be used with `@db.ObjectId` to generate an `ObjectId` in MongoDB)
+  - [`cuid()`](#cuid)
+  - [`uuid()`](#uuid)
+  - [`now()`](#now)
+
+- Default values are currently not allowed on relation fields in the Prisma schema.
```

Comment:
Unrelated to PR: This should have an issue that can be linked here.

Reply:
https://github.com/prisma/docs/issues/2021